components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/URITemplate.java

```diff
@@ -448,4 +449,30 @@ public List<OperationPolicy> getOperationPolicies() {
     public void addOperationPolicy(OperationPolicy policy) {
         operationPolicies.add(policy);
     }
+
+    public String getThrottlingLimit() {
+        return throttlingLimit;
+    }
+
+    public void setThrottlingLimit(String throttlingLimit) {
+        String convertedThrottlingLimit;
+        switch (throttlingLimit) {
+            case "10KPerMin":
```

Comment:
Covert to hold JSON format.

Reply:
fixed with https://github.com/wso2/carbon-apimgt/pull/11760/commits/adc2699be486a1efcf890afd6dbe203791b3363f